cpp/benchmarks/CMakeLists.txt

```diff
@@ -23,8 +23,17 @@ target_compile_options(
 )
 
 target_link_libraries(
-  cudf_datagen PUBLIC GTest::gmock GTest::gtest GTest::gmock_main GTest::gtest_main
-                      benchmark::benchmark nvbench::nvbench Threads::Threads cudf cudftestutil
+  cudf_datagen
+  PUBLIC GTest::gmock
+         GTest::gtest
+         GTest::gmock_main
+         GTest::gtest_main
+         $<TARGET_NAME_IF_EXISTS:conda_env>
```

Comment:
Can you help me understand why this is `PUBLIC` here but it is `PRIVATE` in `cpp/CMakeLists.txt` and `cpp/tests/CMakeLists.txt`?

Reply:
Whoops, it is `PUBLIC` here on purpose so we don't have to add it to each benchmark as well.

Reply:
I will mark it as `PRIVATE` on `cudf_datagen` and also add it to each benchmark so we have consistency